recipe/meta.yaml

```diff
@@ -1,42 +1,58 @@
+{% set name = "usaddress" %}
 {% set version = "0.5.10" %}
-{% set sha = "1a8ebf62d0cce58d7d8286dde70373c530a9317b6fe1752a4197b75b7d0870e3" %}
 
 package:
-    name: usaddress
-    version: {{ version }}
+  name: {{ name }}
+  version: {{ version }}
 
 source:
-    fn: usaddress-{{ version }}.tar.gz
-    url: https://pypi.io/packages/source/u/usaddress/usaddress-{{ version }}.tar.gz
-    sha256: {{ sha }}
+  # tests and license are not in the tarball on pypi
+  url: https://github.com/datamade/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: ef83b69900fe490510fff6aa7239eef1b6a4ea4d00482c73ed4e75096da20f72
 
 build:
-    number: 1001
-    noarch: python
-    script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
```

Comment:
It's generally recommended to not change the build number if we don't change the version number to avoid having the conda-forge package win over our package if a user uses both conda-forge and defaults.

In this case it's not a big deal because it's not going on defaults. More something to keep in mind for future packages.

Reply:
the huge build number is weird too. I couldn't find any info on conda-forge about why. I guess that doesn't change what you said, though.

Reply:
ok thanks, may be valid in this case: if SF also uses CF and SF channels

Reply:
restored

Reply:
thank you for the info, restore to `1001`, in case SF will use both CF and SF channels